From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jocelyn Turcotte <jocelyn.turcotte@digia.com>
Date: Wed, 27 Nov 2013 13:31:28 +0100
Subject: Blacklist the skypebuttons plugin

Even though plugins are disabled it happens that the libraries are
loaded and cause a crash since skypebuttons pulls Qt4 with it on
Linux and creates a dynamic linking error conflicting with Qt5
symbols.

Add the plugin library name to the blacklist to avoid it being
loaded altogether.
We have a similar fix in QtWebKit doing the same thing.

Change-Id: I6706d45d1f19252d3b60af7b97c0a3a729dfe8b7
Reviewed-by: Andras Becsi <andras.becsi@digia.com>
---
 content/common/plugin_list_posix.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/content/common/plugin_list_posix.cc b/content/common/plugin_list_posix.cc
index 3e1353be..7c04485 100644
--- a/content/common/plugin_list_posix.cc
+++ b/content/common/plugin_list_posix.cc
@@ -151,6 +151,7 @@ bool IsUndesirablePlugin(const WebPluginInfo& info) {
 // http://code.google.com/p/chromium/issues/detail?id=38229
 bool IsBlacklistedPlugin(const base::FilePath& path) {
   const char* kBlackListedPlugins[] = {
+    "skypebuttons.so",    // Crash on dlopen since it uses Qt4
     "nppdf.so",           // Adobe PDF
   };
   std::string filename = path.BaseName().value();
